src/adapters/postgres/user-adapter.ts

```
@@ -1419,12 +1534,12 @@ export class PostgresUserService implements IServicelocator {
 
     //Validation for duplicate fields
     if (duplicateFieldKeys.length > 0) {
-      return (error = `Duplicate fieldId detected: ${duplicateFieldKeys}`);
+      return (error = API_RESPONSES.DUPLICATE_FIELD(duplicateFieldKeys));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)